Doc/library/importlib.rst

```diff
@@ -1006,6 +1006,14 @@ find and load modules.
       :exc:`ImportError` is raised.
 
 
+.. class:: NamespacePath(name, path, path_finder)
```

Comment:
The documentation explains what *name* and *path_finder* is, but it does not explain what *path* is.